### PR TITLE
Add actions for Pods

### PR DESCRIPTION
**Pull request by KKoukiou**
Screen Shot 2020-08-05 at 09 35 16

### CODE REVIEW

File: src/Containers.jsx [hunk 1]
```diff
@@ -485,6 +488,10 @@ class Containers extends React.Component {
                                                 <div className='pod-name'>{caption}</div>
                                                 <div>{_("pod group")}</div>
                                             </h4>
+                                            <div className='panel-actions'>
```
Comment (marusak) [inline src/Containers.jsx:491]: Badge is cool, of course, just maybe it should be the same for container statu as well then?

File: lib/table.css [hunk 2]
```diff
@@ -0,0 +1,271 @@
+/*
+ * Keep in sync with https://github.com/cockpit-project/cockpit/tree/master/pkg/lib/table.css
```
Comment (marusak) [inline lib/table.css:2]: I prefer to not to write this header - as it makes sync a that bit more difficult :)

Comment (KKoukiou) [reply lib/table.css:2]: ack I plan anyway to make sync automatic through a makefile rule

Comment (marusak) [reply lib/table.css:2]: That would be cool :) I just do `cp ../cockpit/pkg/lib/* lib` and then `git add -u`, but rule that would copy just the existing files would be great :)

File: src/Containers.jsx (hunk 1 above)
Comment (KKoukiou) [reply src/Containers.jsx:491]: Indeed it looks nicer to be consistent. Fixed.

**Comment by KKoukiou** [comment]
@marusak fixed your comments and localized the pod status strings as well.

**Comment by KKoukiou** [comment]
> Thank you, looks great!

oh wait - I just noticed the extra { t: 0 } I pass by mistake in all API calls, I need to remove this - sorry.

**Comment by marusak** [comment]
This is release-noty for sure, but since we are just introducing pods, I think we can just do one entry like "Show pods - Show pods with containers in them and enable stopping/starting and deleting them"

**Comment by KKoukiou** [comment]
> This is release-noty for sure, but since we are just introducing pods, I think we can just do one entry like "Show pods - Show pods with containers in them and enable stopping/starting and deleting them"

Yes, it should be one entry for pods definitely.

**Pull request merged**